src/components/link/link.component.js

```diff
@@ -1,13 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { assign } from 'lodash';
 import { Link as RouterLink } from 'react-router';
 import Icon from '../icon';
-import { validProps } from '../../utils/ether';
 import Event from '../../utils/helpers/events';
-import tagComponent from '../../utils/helpers/tags';
-import { LinkStyle, LinkStyleAnchor } from './link.style';
+import LinkStyle from './link.style';
 import OptionsHelper from '../../utils/helpers/options-helper';
+import tagComponent from '../../utils/helpers/tags/tags';
```

Comment:
I think this can be simplified to:
```suggestion
import tagComponent from '../../utils/helpers/tags';
```

Reply:
Thanks for committing this fix in 9b51a45 :thumbsup: